languages/fsharp/exercises/concept/records/.docs/instructions.md

```diff
@@ -0,0 +1,107 @@
+In this exercise you're a big sports fan and you've just discovered a passion for NBA basketball. Being new to NBA basketball, you're doing a deep dive into NBA history, keeping track of teams, coaches, their win/loss records and comparing them against each other.
+
+As you don't yet have a favorite team, you'll also be developing an algorithm to figure out whether to root for a particular team.
+
+You have seven tasks to help you develop your proprietary _root-for-a-team_ algorithm.
+
+### 1. Define the model
```

Comment:
I feel like this could be exploded out into the next 3 tasks. I sort of feel like you could lose it altogether - I think there's enough info in 2/3/4 without this.

Reply:
I can't do that, as without having the record types defined, the code won't compile.